style.css

```diff
@@ -409,15 +385,107 @@ h2 {
   color: #7f8cff;
 }
 
-.icon {
+.form-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url(./images/background/Contactcolor.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 779px;
+}
+
+.form-section > h2 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  color: #fff;
+  width: 801px;
+  text-align: center;
+  margin-top: 120px;
+  margin-bottom: 20px;
+}
```

Comment:
- [ ] Please fix the display of the design in the mobile version. You can clearly see in the table below that there is an extra spacing on the right, which comes from this class which has fixed width `801px` which is too large on mobile view.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/161175304-9df0482a-d073-4ddd-8610-1637f680ca7e.png) | ![image](https://user-images.githubusercontent.com/62290545/161175396-02babecf-6282-4635-aa03-2257a4de509e.png) |

Reply:
I did not get your point, should I make the width with 801px, or should I change the width.